source/NVDAObjects/__init__.py

```diff
@@ -316,6 +316,17 @@ def findOverlayClasses(self, clsList: typing.List[typing.Type["NVDAObject"]]) ->
 
 	beTransparentToMouse = False  #:If true then NVDA will never consider the mouse to be on this object, rather it will be on an ancestor.
 
+	def objectFromPointRedirect(self, x: int, y: int) -> "NVDAObject | None":
+		"""Redirects NVDA to another object if this object is retrieved from on screen coordinates.
+		:param x: the x coordinate.
+		:type x: int
+		:param y: the y coordinate.
+		:type y: int
+		:return: The object that NVDA should be redirected to.
+		:rtype: L{NVDAObject}
+		"""
```

Comment:
The types are in the function declaration type hints, we don't add them to the docstring

```suggestion
		"""Redirects NVDA to another object if this object is retrieved from on-screen coordinates.
		:param x: the x coordinate.
		:param y: the y coordinate.
		:return: The object that NVDA should be redirected to.
		"""
```